src/pages/index.tsx

```diff
@@ -1,18 +1,11 @@
-import { Text } from "@nextui-org/react";
+// import { Text } from "@nextui-org/react";
+import Hero from "src/components/hero/Hero";
```

Comment:
Use the `@` path alias, see the `tsconfig.json` file & it's [respective docs](https://www.typescriptlang.org/tsconfig#paths) for reference